subprojects/android-test/test-inhouse-runner/src/main/kotlin/com/avito/android/runner/InHouseInstrumentationTestRunner.kt

```diff
@@ -264,7 +267,7 @@ abstract class InHouseInstrumentationTestRunner :
             report.registerIncident(AppCrashException(incident))
             report.reportTestCase()
         } catch (t: Throwable) {
-            Log.e(tag, "Error during reporting test after global exception handling", t)
+            Log.w(tag, "Can't register and report unexpected incident", t)
```

Comment:
We already don't fail a test if had a failure while register unexpected incident
But ERROR logging level led to misunderstanding behavior. So I change it to WARN